packages/jaeger-ui/src/components/SearchTracePage/SearchResults/ScatterPlot.jsx

```diff
@@ -16,20 +16,29 @@ import React, { useRef, useState, useLayoutEffect } from 'react';
 import moment from 'moment';
 import PropTypes from 'prop-types';
 import { XYPlot, XAxis, YAxis, MarkSeries, Hint } from 'react-vis';
-import { compose, withState, withProps } from 'recompose';
 
 import { FALLBACK_TRACE_NAME } from '../../../constants';
 import { ONE_MILLISECOND, formatDuration } from '../../../utils/date';
 
 import './react-vis.css';
 import './ScatterPlot.css';
 
-function ScatterPlotImpl(props) {
-  const { data, onValueClick, overValue, onValueOver, onValueOut, calculateContainerWidth } = props;
+export default function ScatterPlot(props) {
+  const { data, onValueClick, calculateContainerWidth } = props;
 
   const containerRef = useRef(null);
   const [containerWidth, setContainerWidth] = useState(0);
 
+  const [overValue, setValueOver] = useState(null);
+
+  const onValueOver = value => {
```

Comment:
Is there a way to simulate this event in unit tests?

Reply:
I updated `ScatterPlot.test.js` to simulate this event in the unit tests.